vars/runDirectDebitSmokeTest.groovy

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env groovy
+
+def call(String aws_profile = "test", String smoke_tags = "uk.gov.pay.endtoend.categories.SmokeDirectDebitPayments", boolean promoted_env = false) {
```

Comment:
given the name of this function, you don;t need to pass smoke tags in, just hardcode it in the call to runSmokeTest. Same with others

Reply:
oh I see what you've done now. Sorry, I meant have a generic runSmokeTest thing (as you had) and then have specific functions for each of the three types of test that would do eg.

```
runSmokeTest(smoke_tags: "uk.gov.pay.endtoend.categories.SmokeDirectDebitPayments")
```